src/hearings/services/hearings.service.ts

```diff
@@ -52,4 +52,8 @@ export class HearingsService {
   public updateHearingActuals(hearingId: string, hearingActualsModel: HearingActualsModel): Observable<HearingActualsModel> {
     return this.http.put<HearingActualsModel>(`api/hearings/hearingActuals/${hearingId}`, hearingActualsModel);
   }
+
+  public submitHearingActuals(hearingId: string): Observable<any> {
+    return this.http.post<any>(`api/hearings/hearingActualsCompletion/${hearingId}`, {});
```

Comment:
as chat body will be null as the Back end api do not accept any body including {}.

Reply:
@rayliangatos - as discussed I have updated the code to use null.